src/gpuarray_blas_cuda_cublas.c

```diff
@@ -528,6 +763,269 @@ static int dgemv(cb_order order, cb_transpose transA, size_t M, size_t N,
   return GA_NO_ERROR;
 }
 
+static int hgemvBatch(cb_order order, cb_transpose transA,
+                      size_t M, size_t N, float alpha,
+                      gpudata **A, size_t *offA, size_t lda,
+                      gpudata **x, size_t *offX, size_t incX,
+                      float beta, gpudata **y, size_t *offY, size_t incY,
+                      size_t batchCount, int flags) {
+  return GA_DEVSUP_ERROR;
+}
+
+static int sgemvBatch(cb_order order, cb_transpose transA,
+                      size_t M, size_t N, float alpha,
+                      gpudata **A, size_t *offA, size_t lda,
+                      gpudata **x, size_t *offX, size_t incX,
+                      float beta, gpudata **y, size_t *offY, size_t incY,
+                      size_t batchCount, int flags) {
```

Comment:
Why sgemvBatch have a flag attribute that isn't used which sgemmBatch don't have one? What possible futur utility do you see? Idem for all new ?gemvBatch and ?gerBatch method